lib/actual_db_schema.rb

```diff
@@ -5,6 +5,7 @@
 require "csv"
 require_relative "actual_db_schema/git"
 require_relative "actual_db_schema/configuration"
+require_relative "actual_db_schema/console_migrations"
```

Comment:
Please try to find a way to make the module independent and not load by default. It should be included into the Rails console on the fly only if an option enables it.